Data/ByteString/Streaming/Internal.hs

```diff
@@ -349,27 +352,29 @@ unpackBytes bss = dematerialize bss
             loop sentinal (p `plusPtr` (-1)) (Step (x :> acc))
 {-# INLINABLE unpackBytes #-}
 
--- copied from Data.ByteString.Unsafe for compatibility with older bytestring
+-- | Copied from Data.ByteString.Unsafe for compatibility with older bytestring.
 unsafeLast :: B.ByteString -> Word8
 unsafeLast (B.PS x s l) =
     accursedUnutterablePerformIO $ withForeignPtr x $ \p -> peekByteOff p (s+l-1)
  where
       accursedUnutterablePerformIO (IO m) = case m realWorld# of (# _, r #) -> r
 {-# INLINE unsafeLast #-}
 
--- copied from Data.ByteString.Unsafe for compatibility with older bytestring
+-- | Copied from Data.ByteString.Unsafe for compatibility with older bytestring.
 unsafeInit :: B.ByteString -> B.ByteString
 unsafeInit (B.PS ps s l) = B.PS ps s (l-1)
 {-# INLINE unsafeInit #-}
 
--- | Consume the chunks of an effectful ByteString with a natural right fold.
+-- | Consume the chunks of an effectful `ByteString` with a natural right fold.
 foldrChunks :: Monad m => (B.ByteString -> a -> a) -> a -> ByteString m r -> m a
 foldrChunks step nil bs = dematerialize bs
   (\_ -> return nil)
   (fmap . step)
   join
 {-# INLINE foldrChunks #-}
 
+-- | Consume the chunks of an effectful `ByteString` with a left fold. Suitable
+-- for use with `SP.mapped`.
```

Comment:
```suggestion
-- | Consume the chunks of an effectful 'B.ByteString' with a left fold. Suitable
-- for use with 'SP.mapped'.
```

Reply:
In this case we're referring to the *streamed* bytestring, not the chunk type. This confusion will be fixed when I do the mass renaming in the next PR.

Reply:
Ah, yes, wrong module prefix... Well spotted.